spec/helpers/quadicon_helper_spec.rb

```diff
@@ -15,18 +15,42 @@
   end
 
   describe "#render_quadicon_text" do
+    before(:each) do
+      @settings = {:display => {:quad_truncate => "m"}}
+    end
+
     subject { helper.render_quadicon_text(item, row) }
 
     let(:item) do
       FactoryGirl.create(:vm_vmware)
     end
 
     let(:row) do
-      Ruport::Data::Record.new(:id => 10000000000534)
+      Ruport::Data::Record.new(:id => 10_000_000_000_534, "name" => name)
```

Comment:
The id was pre-existing, I added underscores per Rubocop. I think the deal with the ids is that sometimes this helper gets the id from the actual object (item) and sometimes it asks the "row". This is good to know, I'm working on changes upstream and I'll fix this then.